### DISCUSSION TITLE
Stacker News Thunderstorms

### DISCUSSION THREAD

**Discussion by ghost** [stackernews/stacker.news#1866] [Ideas]
**Is your feature request related to a problem? Please describe.**
The donations feature on Stacker News is not widely used, with roughly 4 stackers per day donating sats.

This may be due to a lack of visibility (the donate button is one page removed from the homepage), but another element worth noting is that donators don't get any recognition for their contributions or the immediate gratification of seeing their money going to support other stackers. Currently the funds donated to the rewards pool are sent out at midnight the next day in varying proportions depending on the contributions of other stackers.

**Describe the solution you'd like**
To solve the problems outlined above, Stacker News could replace the existing donations feature with Thunderstorms.

Thunderstorms make donations more visible, with lower latency, and ultimately make the experience of donating sats more fun.

Thunderstorms work as follows:

- When a stacker long-presses the Lightning bolt on any post or comment, they will see a new button below their recommended zap amounts that says "Start a Thunderstorm" or something to that effect
- If they click it, they are prompted to enter a zap amount (1000+ sats)
- That money is then immediately zapped out to everyone who made a post or comment in the last 24 hours
- Higher quality posts and comments will help stackers earn more, as will a higher quantity of posts and comments
- Stackers who receive this zap will immediately get a notification that says "you've been thunderstruck. @insertusername started a thunderstorm and you stacked (insert number) sats for your recent posts and comments" 
- finally, all the lightning bolt icons on the website will flicker randomly for a few minutes after a thunderstorm starts for all stackers to see

**Comment by ekzyis** [comment]
Linking the discussion on SN here: https://stacker.news/items/211193

**Comment by ekzyis** [comment]
> That money is then immediately zapped out to everyone who made a post or comment in the last 24 hours

What is the relation to the item on which the stacker used long press? Just wondering if the "start thunderstorm" should be somewhere else

**Comment by ghost** [comment]
@ekzyis The button could definitely be located elsewhere, and there's no particular reason it needs to be tied to the post a stacker zaps. 

My thinking is just that since we already have a lightning bolt icon, adding a second one somewhere else on the site might be confusing. Hiding it within the existing zap button might help people understand that by clicking it, they're spending sats.
